src/backend/crud/projects.ts

```diff
@@ -8,7 +8,7 @@ import type {
 import type { Response } from '@backend/Types';
 import { getUser } from '@backend/auth';
 import type { ApiPostInviteUserToProject } from 'src/Types';
-import type { InviteListEntry } from '@apps/project-collaboration/InviteListOfUsers/InviteListOfUsers';
+import type { InviteListEntry } from '@apps/project-collaboration/InviteListOfUsersDialog/InviteListOfUsers';
```

Comment:
That file doesn't seem to exist any more. (Should be `InviteListOfUsersDialog`?)

In general, I think I'd bundle the types with the backend code, and then import from there into the UI code, rather than vice versa. But that's really just a matter of taste I guess & no strong opinion either way.